TestOnly/pom.xml

```diff
@@ -0,0 +1,368 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>net.snowflake</groupId>
+  <artifactId>snowflake-jdbc-test</artifactId>
+  <version>3.9.2</version>
```

Comment:
@mknister, this is the JDBC version to tests. So when you need to change the version, you'll just need to update this.

Reply:
Awesome! Thanks